scripts/get_resource_names.go

```diff
@@ -0,0 +1,229 @@
+// get_resource_names collects the resource names from the test files in test-artifacts
+// By default, it skips the names starting with 'Test' or 'test', as they are otherwise
+// recognised and handled during the leftover removal operation.
+// The resulting list has to be added to `remove_leftover_test.go`
+
+// Run the code in ./vcd/test-artifacts, possibly after running `make test-binary-prepare` in the top directory
+// go run get_resource_names.go | vim -
```

Comment:
This is the first Go file in a "scripts" folder, though it is not a script. Feels a bit misplaced. This is probably less because its a to-be-compiled Go application, but more because it's purpose is hard to grasp in isolation. Maybe add a line on in what format it returns the result?

Reply:
The result is described already in the main function, but I added it at the top of the file as well.
As for the misplaced file, it is a standalone program, and can't stay with the rest of the vcd files.
Since it needs to be used in `./vcd/test-artifacts`, it is hosted in `./scripts`, from where it gets copied to the appropriated place during `make test-binary`, same as the other files we use for that purpose.